### PR TITLE
Lin/doc

### PR DESCRIPTION
**Pull request by jinlin-flex**
[Jira Ticket: 
](https://flow360.atlassian.net/browse/FLPY-6)Build and deploy documentation for python client onto https://jinlin-flex.github.io/jinlinflex.github.io/

### CODE REVIEW

File: .github/workflows/deploy-doc.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      run:  make html
+
+    - uses: cpina/github-action-push-to-another-repository@main
+      env:
+        API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_DOC_REPO }}
+      with:
+        source-directory: './docs/_build/html/'
+        destination-github-username: 'jinlin-flex'
+        destination-repository-name: 'jinlinflex.github.io'
+        user-email: lin@flexcompute.com
```
Comment (maciej-flexcompute) [inline .github/workflows/deploy-doc.yml:40]: @lei-flex can we use some Flexcompute account here?

File: .github/workflows/deploy-doc.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+name: Build and Deploy New Documentation
+
+on:
+  push:
+    branches:
+      - lin/doc
```
Comment (maciej-flexcompute) [inline .github/workflows/deploy-doc.yml:6]: on:
  workflow_dispatch:

File: docs/flow360doc/component/flow360_params.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
+flow360 solvers parameters
```
Comment (maciej-flexcompute) [inline docs/flow360doc/component/flow360_params.rst:1]: how are these files generated? Automatically or manually?

Comment (jinlin-flex) [reply docs/flow360doc/component/flow360_params.rst:1]: The rst files are written manually, but the documentation are generated automatically.

**Review by maciej-flexcompute — APPROVED** [review]

**Pull request merged**